components/admin_console/billing/billing_history.tsx

```diff
@@ -151,75 +147,77 @@ const BillingHistory: React.FC<Props> = () => {
-                        </td>
-                        <td className='BillingHistory__table-invoice'>
-                            <a
-                                target='_new'
-                                rel='noopener noreferrer'
-                                href={Client4.getInvoicePdfUrl(invoice.id)}
-                            >
-                                <i className='icon icon-file-pdf-outline'/>
-                            </a>
-                        </td>
+                        </th>
+                        <th className='BillingHistory__table-headerTotal'>
+                            <FormattedMessage
+                                id='admin.billing.history.total'
+                                defaultMessage='Total'
+                            />
+                        </th>
+                        <th>
+                            <FormattedMessage
+                                id='admin.billing.history.status'
+                                defaultMessage='Status'
+                            />
+                        </th>
+                        <th>{''}</th>
                     </tr>
-                ))}
+                    {billingHistory.map((invoice: Invoice) => (
+                        <tr
+                            className='BillingHistory__table-row'
+                            key={invoice.id}
+                        >
+                            <td>
+                                <FormattedDate
+                                    value={new Date(invoice.period_start)}
+                                    month='2-digit'
+                                    day='2-digit'
+                                    year='numeric'
+                                    timeZone='UTC'
+                                />
+                            </td>
+                            <td>
+                                <div>{invoice.current_product_name}</div>
+                                <div className='BillingHistory__table-bottomDesc'>
+                                    <InvoiceUserCount invoice={invoice}/>
+                                </div>
+                            </td>
+                            <td className='BillingHistory__table-total'>
+                                <FormattedNumber
+                                    value={(invoice.total / 100.0)}
+                                    // eslint-disable-next-line react/style-prop-object
+                                    style='currency'
+                                    currency='USD'
+                                />
+                            </td>
+                            <td>
+                                {getPaymentStatus(invoice.status)}
+                            </td>
+                            <td className='BillingHistory__table-invoice'>
+                                <a
+                                    target='_self'
```

Comment:
This is the real change of this PR, setting back the target value to _self so it downloads the invoices in the desktop app